Formula/awsume.rb

```diff
@@ -9,6 +9,11 @@ class Awsume < Formula
   revision 1
   head "https://github.com/trek10inc/awsume.git"
 
+  livecheck do
+    url "https://awsu.me/changelog.html"
+    regex(/\[(\d+(?:\.\d+)+)\] - \d{4}-\d{2}-\d{2} -/i)
```

Comment:
Might be a better and more reliable option, as it seems tags ending in `a` followed by a digit are pre-releases. What do you think?
```suggestion
    url :head
    regex(/v?(\d+(?:\.\d+)+)$/i)
```

Reply:
Nice, I like your idea better

Reply:
I'm reviewing some PRs after the fact and this regex is missing the leading `^`.

@nandahkrishna Be careful when copying the standard snippets to make sure that you don't miss anything. The standard snippet for Git tags like `1.2.3` or `v.1.2.3` is: `/^v?(\d+(?:\.\d+)+)$/i`

Reply:
Sorry about that, didn't realise it. I'll be more careful about it next time 👍🏼